### PR TITLE
Update EN Competition

### PR DESCRIPTION
**Pull request by leonorejf**

### CODE REVIEW

**Comment by leonorejf** [comment]
@TheDaringDan  peux tu double check la traduction

File: en/competition/index.md [hunk 1]
```diff
@@ -7,34 +7,34 @@ ref: competition
 image: "/banner/competition.jpg"
 ---
 
-After participating for many years to the Walking Machine SAE and the Eurobot subsequently, since 2016, we participate to the Robocup@Home international competition.
+After participating for many years in the Walking Machine SAE and the Eurobot competitions subsequently, since 2016, we are competing in the Robocup@Home international competition.
 
-## Robocup@Home in brief
+## Robocup@Home in a few lines
```
Comment (leonorejf) [inline en/competition/index.md]: Robocup-at-Home summarized

File: en/competition/index.md [hunk 2]
```diff
@@ -7,34 +7,34 @@ ref: competition
...
 
-The RoboCup@Home competition is held during the Robocup, the most important competition of Robotics in the world. The latter also presents the Soccer, Rescue and Junior categories. This event held once a year, brings together university and college from across the world.
+The RoboCup@Home is held during the Robocup, the most important robotics competition in the world. The latter also showcase the Soccer, Rescue, and Junior categories. This event, held once a year, brings together university and college from across the world.
```
Comment (leonorejf) [inline en/competition/index.md]: RoboCup-at-Home is held during Robocup, the most important robotics competition in the world. The latter also showcases the Soccer, Rescue, and Junior branches. This event, held once a year, brings together universities and colleges from around the world.

File: en/competition/index.md [hunk 3]
```diff
@@ -7,34 +7,34 @@ ref: competition
...
 
 <center><iframe width="560" height="315" src="https://www.youtube.com/embed/YpjeNa8BAYg" frameborder="0" allowfullscreen></iframe></center><br>
-RoboCup@Home aims to develop robots for personal assistance with the aim of creating the perfect assistant in the tasks of everyday life. To determine the best robots, the teams move to their robot a series of tests to measure their skills and performance in an environment simulating a House. The main axes of development are: Human-machine interaction and cooperation, navigation and mapping in dynamic environment, computer vision and recognition of objects under different light, object manipulation and artificial intelligence.
+RoboCup@Home aims to develop robots for personal assistance with the aim of creating the perfect assistant in everyday tasks. To find the best robots, the teams make their robot do a series of tests to measure their skills and performance in an house like environment. The main fields of development are: Human-machine interaction and cooperation, navigation and mapping in dynamic environment, computer vision and recognition of objects under different light, objects manipulation and artificial intelligence.
```
Comment (leonorejf) [inline en/competition/index.md]: - measure their skills and performance in an domestic environment.
 - human-machine interaction - human en minuscule
 - dynamic environmental navigation and mapping
 - object recognition

File: en/competition/index.md [hunk 4]
```diff
@@ -7,34 +7,34 @@ ref: competition
...
 
-## Déroulement de la compétition
+## During the competition
 
 <center><iframe width="560" height="315" src="https://www.youtube.com/embed/-oGeU-58wrQ" frameborder="0" allowfullscreen></iframe></center><br>
-The events are presented in the form of scenario spread out over two levels. Thus, After passing the qualification phase, the robots are confronted with scenarios of the level 1. The robots having the most points access level 2. The five teams who have accumulated the most points during the scenarios past access to the final round. The winning robot is the one who has accumulated the most points throughout his career.
+The events are presented in the form of scenarios spread out over two levels of difficulty. Thus, after passing the qualification phase, the robots are confronted with level 1 scenarios. The robots that scored the most access level 2. The five teams who have accumulated the most points during the past scenarios access to the final round. The winning robot is the one who has accumulated the most points throughout the different heats.
```
Comment (leonorejf) [inline en/competition/index.md]: The five teams who have accumulated the most points during the competition go on to the final round. The winning robot is the one who has accumulated the most points throughout the different stages.

File: en/competition/index.md [hunk 5]
```diff
@@ -7,34 +7,34 @@ ref: competition
...
 
-## Les scénarios de l'année 2015-2016
+## Last year s scenarios
```
Comment (leonorejf) [inline en/competition/index.md]: Last year's scenarios

File: en/competition/index.md [hunk 6]
```diff
@@ -7,34 +7,34 @@ ref: competition
...
 
-#### Niveau I
+#### Level I
 * Follow a person
 * Clean a room
-* Recognition and association of persons
+* Facial recognition and facial association
```
Comment (leonorejf) [inline en/competition/index.md]: Facial recognition and association

**Review by leonorejf — COMMENTED** [review]
Daniel: Voici quelques changements a amener

**Review by gregoirefra — CHANGES_REQUESTED** [review]
Les changements ont été fait dans les commentaires du thread de la pull request. Merci de les mettres dans le fichier et de commit en conséquence pour que je puisse approve les changements

**Pull request merged**